remixer_example/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I had to add a dummy version, otherwise the build fails if you apply plugin com.google.gms.google-services.

I need it to build even if it doesn't connect to firebase.